tutorials/000_introduction_test.go

```diff
@@ -22,7 +22,7 @@ to ship a set of real tutorials to get people started with this special
 little library but I just couldn't find the right medium to ship a stable,
 deployable, and accessible tutorial series until today.
 
-This book is built into Poly's source code itself. Each tutorial a series
+This book is built into Poly's source code itself. Each tutorial is a series
```

Comment:
I believe this was also in the last push you submitted @soypat? Do you know what's going on here?

Reply:
Git mystifies me. This tends to happen to me sometimes. I can try squashing commits, or that step can be done via github. As you'd prefer

Reply:
Whatever is easiest! @soypat